### ISSUE TITLE
Help testing new branch wrapper

### ISSUE THREAD

**Issue by felquis** [pagarme/react-material-design-icon-components#2]
> Letter from @derekstavis 

Hey guys, an update about material icons:

I have made some changes to the API and now you can import an icon wrapper directly.

Before:

```jsx
import { IconBluetooth48 } from 'react-material-design-icon-components'

...

render () {
  return <IconBluetooth48 />
}
```

After:

```jsx
import { IconBluetooth } from 'react-material-design-icon-components'

...

render () {
  return <IconBluetooth size={48} />
}
```

Using this wrapper brings two main considerations:

* When importing an icon wrapper, all sizes will be imported, even when not used
* If an icon matching `size` isn't available, the nearest size will be chosen

Please test [`wrapper`](https://github.com/pagarme/react-material-design-icon-components/tree/wrapper
) importing icons via wrapper and give feedback about how/if the interface is working for you.

Thanks!

**Comment by felquis** [comment]
Testing with

```
"react-material-design-icon-components": "derekstavis/react-material-design-icon-components#wrapper"
```

**Comment by felquis** [comment]
@derekstavis I still can use the old version, wouldn't be better to remote it to avoid future deprecations? 

```jsx
import icons from 'react-material-design-icon-components'

console.log(icons)
```

![screenshot from 2016-12-08 19-01-48](https://cloud.githubusercontent.com/assets/736728/21027648/ee8b84e2-bd78-11e6-8c24-831332066c6a.png)

**Comment by felquis** [comment]
The new API works great for me

```jsx
<IconAccessible size={48} />
<IconAccessible size={24} />
```
Can we set some proptypes to ensure only 48, 24 too?

**Comment by MarcoWorms** [comment]
I'm gonna kick the bucket here @derekstavis , if we are going to remove the numbers why not remove `Icon` from every component? It's really verbose :P

**Comment by derekstavis** [comment]
Yeah. I thought about removing the `Icon` prefix, but some icons start with numbers, as in `3dRotation`, which can't be in front of variable names. Also, icons could have some clashes with user-named components, and prefixing with `Icon` solves this issue. Another idea is to use `Ic`, as Google uses.

**Comment by felquis** [comment]
`Icon` works for me.

**Comment by derekstavis** [comment]
Should this one be merged? Does the interface makes sense? @felquis @MarcoWorms
